### ISSUE TITLE
Allow for Template Extensions to Easily Inline Code

### ISSUE THREAD

**Issue by mankyd** [miracle2k/webassets#246]
Currently, there is no quick and easy way to inline CSS and/or JS via webassets. For instance, the Jinja2 extension supports the following usage:

```
{% assets output="css/m.%(version)s.css", "css/a.css", "css/b.css" %}
<link rel="stylesheet" href="{{ ASSET_URL }}">
{% endassets %}
```

However, if you want to inline that css instead, you're out of luck. It would be great if I could easily replace that with:

```
{% assets filters="cssmin", "css/a.css", "css/b.css" %}
<style>{{ ASSET_CONTENTS }}</style>
{% endassets %}
```

**Comment by zllak** [comment]
+1. That is kind of a must have feature. Any plan on merging this @miracle2k ? Thanks :)

**Comment by miracle2k** [comment]
Yes, but but the issue noted in #384 that would need to be addressed first.

**Comment by zllak** [comment]
@miracle2k what is blocking this issue right now ? I might have some time to dig. I really need this to go further :)

**Comment by miracle2k** [comment]
As noted on the PR, the pull request forces a file system touch currently, even if ASSETS_CONTENT is not used, and auto_build is False. Part of the promise of auto_build=False is that the filesystem is not touched, and in fact now files exists; the urls could be generated solely based on the manifest.

**Comment by joehand** [comment]
:thumbsup:
